.github/workflows/make-live.yml

```diff
@@ -0,0 +1,68 @@
+name: Make Live
+# Make live versions of .Rmd files in training modules
+
+# This workflow only runs when it is manually dispatched
+on:
+  push:
+    branches: [ cansavvy/gha-make-live ]
+  workflow_dispatch:
+    inputs:
+      rendering:
+        description: Should rendering be completed? TRUE or FALSE
+        required: true
+        default: 'FALSE'
+
+jobs:
+  # This workflow contains a single job called "build-all"
+  make-live:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    container:
+      image: ccdl/training_dev:latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          # get the full repo
+          fetch-depth: 0
+          # use alexslemonade-docs-bot
+          token: ${{ secrets.DOCS_BOT_GITHUB_TOKEN }}
+
+      # Add a download data step here
+      #- name: Download data
+
+      # Run the make-live.R script
+      - name: Run make-live.R
+        id: make_live
+        run: Rscript --vanilla scripts/make-live.R --rendering ${{ github.event.inputs.rendering }}
```

Comment:
to test, you will need to replace this temporarily, since `github.event.inputs.rendering` doesn't get set except in the `workflow_dispatch` trigger
```suggestion
        run: Rscript --vanilla scripts/make-live.R --rendering FALSE
```

Reply:
I ran make-live.R outside of the workflow and it works so I think this other part makes sense to test after it's merged and we can use it in manual dispatch?

Reply:
I'm willing to try that.

Reply:
Remove the `push:` trigger before merging...